spec/unit_tests/moab/storage_object_spec.rb

```diff
@@ -493,7 +493,7 @@
       end
 
       it 'does not raise error' do
-        expect { storage_obj.reconstruct_version(1, bag_dir) }.not_to raise_error(Errno::EEXIST)
+        expect { storage_obj.reconstruct_version(1, bag_dir) }.not_to raise_error
```

Comment:
RSpec told me to.

Reply:
So, you just do everything your told? :-P

Reply:
literally everything